processing_code/01knit-RMD-2-MD_NDSRepo.R

```diff
@@ -16,13 +16,14 @@ dirs <- c("R/eddy4r/",
           "R/download-explore/",
```

Comment:
Try to avoid pushing this file - it frequently creates merge conflicts with others given different Git & data files paths, etc. Thanks.

Reply:
Weird, it's in my gitignore, I thought it was taken care of. Sorry!

Reply:
If you can figure out how to get it to "predominately" stick in a gitignore I'd love it, and tried to do that myself – however, I believe the problem is files can only be ignored using gitigore if they are not already part of the repository.  We do want this file to be part of the repository as we need new users forking the repo to get it and the ability to update the file and have all users get the updates when there are any structural or code changes. If anyone can think of another solution, I'm all ears.

Reply:
Got it. I'll poke around a bit and see if I can find a solution.

Reply:
Ahhh ok, I have been dealing with the same thing. Curious to see if there is an answer to this (like a .git.ignore.updates option)